srcpkgs/starship/template

```diff
@@ -3,7 +3,7 @@ pkgname=starship
 version=0.44.0
 revision=1
 build_style=cargo
-build_helper="rust"
+build_helper=qemu
```

Comment:
Does this really not require the rust build helper? You can specify more than one.

Reply:
It's added by the cargo build style, so it doesn't need to be repeated.